src/nodes/ContainerNode.js

```diff
@@ -43,13 +40,16 @@ export default class ContainerNode {
     };
     this.stream = stream;
 
+    this.canvasSize = {
```

Comment:
This can change, no?

Reply:
Yes, but I don't care about it right now. We can handle it later.